### DISCUSSION TITLE
Event AfterRenderOptions

### DISCUSSION THREAD

**Discussion by Sempervivum** [brianvoe/slim-select#507] [Ideas]
Hi,
I'm supporting a developer who intends to select a default option at startup. As the data source is remote, fetching is async and we need an event when data is fetched and options are ready. I added this to the JS and named the function afterRenderOptions.
Take a look if I did it correctly.
Best regards, Ulrich
[slimselect-mod.txt](https://github.com/brianvoe/slim-select/files/13929211/slimselect-mod.txt)

**Comment by brianvoe** [comment]
Sorry Im not sure what to do with this. If you would like to submit a pr please do so for any new features.
